src/commands.ts

```diff
@@ -22,9 +23,34 @@ export function init(context: ExtensionContext) {
     commands.registerCommand(
       'diffMerge.compareFileWithClipboard',
       compareFileWithClipboard
-    )
+    ),
+    commands.registerCommand('diffMerge.openWithDiffMerge', reopenCurrentWithDiffMerge),
   );
 
+  async function reopenCurrentWithDiffMerge() {
+    const editor = window.activeTextEditor;
+    if (!editor) return;
+    const visibleEditors = window.visibleTextEditors;
+    const editorIndex = visibleEditors.indexOf(editor);
+    // This might not be always true. But it's the only way I can think of how to know whether it's a diff-editor.
+    const isDiffEditor = (editor: TextEditor) => editor.viewColumn === undefined && ['git', 'file'].includes(editor.document.uri.scheme);
```

Comment:
If it's a `git` diff view, it should goes through the [git diff process](https://github.com/moshfeu/vscode-diff-merge/blob/b82b059c1958abecdf4489a20c1d4fd1dc9d5277/src/commands.ts#L76) to get the content from git, not from a physical file.

Reply:
Ok, so I simply call your own api with the paths.

Reply:
Done.